### ISSUE TITLE
Add block button outside of keyboard navigation

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/tobira#647]
On the “Edit page” view, the add block button has no tab index and therefor cannot be tabbed to using a keyboard. This also means that it may not be picked up by a screen reader:

![Screenshot from 2023-01-05 19-30-29](https://user-images.githubusercontent.com/1008395/210855918-2d1f2c7f-541c-4795-9004-3797834215c6.png)
